### PR TITLE
Quick start for existing users v2: delete edit your homepage tour 

### PR DESCRIPTION
**Pull request by AjeshRPai**
Fixes Issue - https://github.com/wordpress-mobile/WordPress-Android/issues/16782
Parent PR - https://github.com/wordpress-mobile/WordPress-Android/pull/16790

This PR removes Edit your home page tour from the FluxC Store

To test 
Testing instructions are available on https://github.com/wordpress-mobile/WordPress-Android/pull/16790

### CODE REVIEW

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.kt [hunk 1]
```diff
@@ -1869,6 +1869,10 @@ open class WellSqlConfig : DefaultWellConfig {
                 175 -> migrate(version) {
                     db.execSQL("ALTER TABLE PostModel ADD ANSWERED_PROMPT_ID INTEGER")
                 }
+                176 -> migrate(version) {
+                    db.execSQL("DELETE FROM QuickStartTaskModel WHERE TASK_NAME='edit_homepage' " +
+                        "AND TASK_TYPE='grow';")
```
Comment (ashiagr) [inline fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.kt]: ⚠️ Need to replace `TASK_TYPE` `grow`  with `customize` as `edit_homepage` task  is of `customize` type.

**Review by ashiagr — CHANGES_REQUESTED** [review]
Found one blocking issue. I've added details in the corresponding `WPAndroid` PR on how it can cause `Quick Start` to be completed even when a task is incomplete.

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.kt (hunk 1 above)
Comment (AjeshRPai) [reply fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.kt]: @ashiagr  I am sorry. Silly mistake. Thanks for pointing it out.  I have updated the task type in the commit [08115e5](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2446/commits/08115e553dcf3440a807d7df1d0d84e9e54e1c83). Once the CI is run successfully I will update the fluxc Version in the parent PR. I will request a review after that for both the PR's.

**Review by ashiagr — APPROVED** [review]
Thank you for the changes, all looks good to me!

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.kt (hunk 1 above)
Comment (ashiagr) [reply fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.kt]: No worries. It happens to all of us. All looks good now. 🎉

**Pull request merged**